### ISSUE TITLE
Wrong formatting with comments.

### ISSUE THREAD

**Issue by Shu-Ji** [sqls-server/sqls#122]
```mysql
-- comment 111
SELECT * from tb1;
-- comment 222
SELECT * from tb2;
```

Formatting Request:
```json
{
    "jsonrpc": "2.0",
    "method": "textDocument/formatting",
    "params": {
        "textDocument": {
            "uri": "file:///workspace/mysql.sql"
        },
        "options": {
            "tabSize": 4,
            "insertSpaces": true
        }
    },
    "id": 1703824210
}
```
Original code:
<img width="305" alt="image" src="https://github.com/sqls-server/sqls/assets/1127594/b1933011-007c-42ad-b4c0-b25936f3eed8">

lsp-server result:
```json
{
    "id": 1703824210,
    "result": [
        {
            "range": {
                "start": {
                    "line": 0,
                    "character": 14
                },
                "end": {
                    "line": 3,
                    "character": 18
                }
            },
            "newText": "SELECT\n    *\nFROM\n    tb1;SELECT\n        *\n    FROM\n        tb2;"
        }
    ],
    "jsonrpc": "2.0"
}
```

Formatted code:
<img width="256" alt="image" src="https://github.com/sqls-server/sqls/assets/1127594/6eee3964-0118-4650-b3e7-f3ff6b11ebf9">

Problem:
1. Comments were removed.
2. The second `SELECT` is on wrong line.

**Comment by mattn** [comment]
@lighttiger2505 from my understanding, sqls does not support multiple SQL statements. right?

**Comment by Shu-Ji** [comment]
> ```sql
> -- comment 111
> SELECT * from tb1;
> ```

Even one statement will not work:
```sql
-- comment 111
SELECT * from tb1;
```

**Comment by Shu-Ji** [comment]
> @lighttiger2505 from my understanding, sqls does not support multiple SQL statements. right?

Even one statement will not work:
```sql
-- comment 111
SELECT * from tb1;
```

**Comment by mattn** [comment]
addressed in https://github.com/sqls-server/sqls/pull/123

**Comment by mattn** [comment]
fixed in #123
